### PR TITLE
opnode: Add Dockerfile/Makefile

### PR DESCRIPTION
**Pull request by mslipper**
- Adds a Dockerfile so that we can build opnode images
- Adds a Makefile to make building the opnode easier

### CODE REVIEW

**Comment by trianglesphere** [comment]
One longer term Q is that given Conner added a makefile in `bss` in #179 (and there's another one in `opnode/contracts`) do we want a makefile at the root of the repo or in the root of each project?

**Comment by maurelian** [comment]
However we manage make files, a single command to build everything would be nice. That can be a separate PR tho.

File: Makefile [hunk 1]
```diff
@@ -0,0 +1,7 @@
+opnode:
+	go build -o ./bin/op ./opnode/cmd
+.PHONY: opnode
+
+clean:
+	rm -rf ./bin
+.PHONY: clean
```
Comment (maurelian) [inline Makefile:7]: should add the blank line, then LGTM.

**Review by maurelian — APPROVED** [review]

**Comment by norswap** [comment]
Agree with @trianglesphere we should consolidate the Makefiles in an upcoming PR.

**Pull request merged**